kotlinpoet/src/commonMain/kotlin/com/squareup/kotlinpoet/Closeable.kt

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright (C) 2024 Square, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.squareup.kotlinpoet
+
+public expect interface Closeable : AutoCloseable
```

Comment:
We should use `AutoCloseable` on all platforms and only add the JVM's `Closeable` where needed. We don't want to put a type in our public API for just doing this, though.

Reply:
I didn't realize that the types that implement `Closeable` are internal types, you're right, it doesn't seem necessary to define an additional `Closeable`